index.html

```diff
@@ -112,27 +112,31 @@ <h5 class="cardtitle fw-bolder mx-2 text-center opacity-100">Network</h5>
       </div>
 
       <div class="card register-btn h-50 p-2 mx-auto d-flex flex-row justify-content-evenly align-items-center
-                rounded-0 mb-3">
-        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light text-decoration-none
+                rounded-0  my-4 ">
+        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light
           " href="#">Join Our Art Exhibition</a>
       </div>
       
     </section>
 
-    <section class="mt-4 mb-5 w-100 bg-light">
+    <section class="mt-4 mb-2 w-100 speaker-details">
       <div class="title d-flex flex-column justify-content-center">
         <h3 class="text-dark h4 fw-bold mt-4  text-center">Featured Speakers</h3>
         <hr class="hr mx-auto border mb-3 border-1 border-danger">
       </div>
 
       <!-- dynamic speakers data from app.js -->
-      <div class="speakers d-flex flex-column flex-md-row justify-content-between mx-md-auto px-md-5
-        mt-5 flex-md-wrap " id="speakers-section"></div>         
+      <!-- mobile version -->
+      <div class="speakers h-100 d-flex d-md-none flex-column flex-md-row justify-content-center mx-md-auto px-md-5
+        mt-2 flex-md-wrap pt-4" id="speakers-section"></div>
+      <!-- desktop version -->
+      <div class="speakers d-none d-md-flex gap-1 flex-column flex-md-row justify-content-center mx-md-auto px-md-5
+        mt-5 flex-md-wrap " id="desktop-speakers-section"></div>
     </section>
```

Comment:
- [ ] Great job implementing the `More Button` even though it was an optional requirement :wink: However, as the previous reviewer mentioned, that feature was only present in the mobile version. So kindly remove it when in desktop view and show all the speakers as shown in the original Design :100: 
- [ ] I also noticed that you're still missing the chess background shape for the speakers' avatar pictures and the small gray horizontal line between the contents. Kindly implement these changes as well as shown in the original design. :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/225447825-a0ee4f15-8fd7-418e-bb52-cfb6be58a451.png)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/225447935-b9bf4de1-3abb-417c-bcc8-f9d010068460.png)